src/corefx/System.Globalization.Native/localeNumberData.cpp

```diff
@@ -391,3 +462,44 @@ extern "C" int32_t GetLocaleInfoInt(const UChar* localeName, LocaleNumberData lo
 
 	return UErrorCodeToBool(status);
 }
+
+/*
+PAL Function:
+GetLocaleInfoInt
+
+Obtains integer locale information
+Returns 1 for success, 0 otherwise
+*/
+extern "C" int32_t GetLocaleInfoGroupingSizes(const UChar* localeName, LocaleNumberData localeGroupingData, int32_t* primaryGroupSize, int32_t* secondaryGroupSize)
+{
+	Locale locale = GetLocale(localeName);
+	if (locale.isBogus())
+	{
+		return UErrorCodeToBool(U_ILLEGAL_ARGUMENT_ERROR);
```

Comment:
Why not just return false or 0 here?  UErrorCodeToBool just returns 0 for non-successful error codes.